<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controll</class>
 <widget class="QMainWindow" name="Controll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controll</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>248</width>
      <height>183</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="lvClerks">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCompute">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчитать ЗП для  сотрудника</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbComputeAll">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчитать ЗП для всех</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUpdate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Upd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
